app/src/main/java/com/edulectronics/tinycircuit/Controllers/WireController.java

```diff
@@ -13,10 +13,10 @@
  */
 
 public class WireController {
-    private Component first;
+    public Component first;
```

Comment:
waarom is deze public? Als er geen goeie reden voor is dan zo veel mogelijk private houden